### DISCUSSION TITLE
Nushell integration

### DISCUSSION THREAD

**Discussion by fdncred** [ajeetdsouza/zoxide#180] [General]
This is not an issue. I just wanted to say thanks. I've been using the latest `main` for a few days and it's working wonderfully!!! Just passing much love to @ajeetdsouza and all the folks who worked on the nushell integration from this thread https://github.com/ajeetdsouza/zoxide/issues/69!

This is how my `config.toml` prompt looks.
```
prompt = "__zoxide_hook; construct_prompt"
```

`construct_prompt` [is from here](https://github.com/nushell/nu_scripts/blob/main/prompt/left_and_right_prompt.nu) if anyone is interested. I'm running it in Windows 10.
![image](https://user-images.githubusercontent.com/343840/114413753-e1657880-9b73-11eb-8c87-1c8f89cf78d3.png)

**Comment by ajeetdsouza** [comment]
Hey @fdncred, I'm glad zoxide is working well for you. Nushell integration would not have been possible without your consistent efforts (both on zoxide and Nushell), so thank you for all your work on this!
